examples/prompting/Chain_of_thought_prompting.ipynb

```diff
@@ -0,0 +1,223 @@
+{
```

Comment:
<blockquote> Note the multiple steps implemented within the prompt.</blockquote><p> </p><p>This example gives fewer steps than the base prompt.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/164/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>